pkg/controller/account/account_controller.go

```diff
@@ -242,143 +243,243 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
+					_, err = awsSetupClient.DeleteAccessKey(&iam.DeleteAccessKeyInput{AccessKeyId: accessKey.AccessKeyId, UserName: &userName})
+					if err != nil {
+						reqLogger.Error(err, "Failed to delete root access keys")
+						return reconcile.Result{}, err
+					}
+				}
 			}
 
-			// Build Aws Account
-			awsAccountID, err = r.BuildAccount(reqLogger, awsSetupClient, currentAcctInstance)
+			// Create new root access keys
+			userSecretInfo, err := CreateUserAccessKey(awsSetupClient, userName)
 			if err != nil {
+				failedToCreateUserAccessKeyMsg := fmt.Sprintf("Failed to create IAM access key for %s", "root")
+				reqLogger.Info(failedToCreateUserAccessKeyMsg)
 				return reconcile.Result{}, err
 			}
 
-			totalaccountwatcher.TotalAccountWatcher.Total += 1
+			// Delete original root access key
+			_, err = awsSetupClient.DeleteAccessKey(&iam.DeleteAccessKeyInput{AccessKeyId: &accessKeyID, UserName: &userName})
+			if err != nil {
+				reqLogger.Error(err, "Failed to delete root access keys")
+				return reconcile.Result{}, err
+			}
+
+			accountClaimSecret.Data =
+				map[string][]byte{
+					"aws_user_name":         []byte(*userSecretInfo.AccessKey.UserName),
+					"aws_access_key_id":     []byte(*userSecretInfo.AccessKey.AccessKeyId),
+					"aws_secret_access_key": []byte(*userSecretInfo.AccessKey.SecretAccessKey),
+				}
 
-			// set state creating if the account was able to create
-			SetAccountStatus(reqLogger, currentAcctInstance, "Attempting to create account", awsv1alpha1.AccountCreating, "Creating")
-			err = r.Client.Status().Update(context.TODO(), currentAcctInstance)
+			err = r.Client.Update(context.TODO(), accountClaimSecret)
 			if err != nil {
+				reqLogger.Error(err, "Failed to update root access keys")
 				return reconcile.Result{}, err
 			}
 
-			// update account cr with awsAccountID from aws
-			currentAcctInstance.Spec.AwsAccountID = awsAccountID
-			err = r.Client.Update(context.TODO(), currentAcctInstance)
+			currentAcctInstance.Status.State = AccountCreating
+			err = r.statusUpdate(reqLogger, currentAcctInstance)
 			if err != nil {
 				return reconcile.Result{}, err
 			}
+			return reconcile.Result{}, nil
+		}
 
-		} else {
+	} else {
+		// Normal account creation
+		// We expect this secret to exist in the same namespace Account CR's are created
+		awsSetupClient, err := awsclient.GetAWSClient(r.Client, awsclient.NewAwsClientInput{
+			SecretName: controllerutils.AwsSecretName,
+			NameSpace:  awsv1alpha1.AccountCrNamespace,
+			AwsRegion:  "us-east-1",
+		})
+		if err != nil {
+			reqLogger.Error(err, "Failed to get AWS client")
+			return reconcile.Result{}, err
+		}
 
-			awsAccountID = currentAcctInstance.Spec.AwsAccountID
+		// Test PendingVerification state creating support case and checking for case status
+		if currentAcctInstance.Status.State == AccountPendingVerification {
+			// reqLogger.Info("Account in PendingVerification state", "AccountID", currentAcctInstance.Spec.AwsAccountID)
```

Comment:
Uncomment this?

Reply:
I was moving this existing code around, so I'm not sure who commented it out or for what reason. I think it would be better to just remove